federation/pkg/federation-controller/replicaset/replicasetcontroller.go

```diff
@@ -329,6 +332,23 @@ func (frsc *ReplicaSetController) schedule(frs *extensionsv1.ReplicaSet, cluster
 	for clusterName, replicas := range overflow {
 		result[clusterName] += replicas
 	}
+	if glog.V(2) {
+		buf := bytes.NewBufferString(fmt.Sprintf("Schedule - ReplicaSet: %s/%s\n", frs.Namespace, frs.Name))
+		sort.Strings(clusterNames)
+		for _, clusterName := range clusterNames {
+			cur := current[clusterName]
+			target := scheduleResult[clusterName]
+			fmt.Fprintf(buf, "%s: current: %3d target: %3d", clusterName, cur, target)
```

Comment:
Why "%3d"? 
Personally, I would just write ""%d", but if you really want an integer, probably something bigger, e.g. %5d?